### ISSUE TITLE
ch07 Client Side Handlebars; handlebars.compile() error

### ISSUE THREAD

**Issue by ShahidYousuf** [EthanRBrown/web-development-with-node-and-express#90]
error : nurseryRhymeTemplate is not a function.
When clicked on "Generate nursery rhyme" the browser throws this error and nothing is rendered.
All site files same as stated in the book.

**Comment by nhs-webleym** [comment]
I'd refer to the file structure on git.  As its not 100% clear from the book what goes where. 
I had similar issues.

**Comment by ShahidYousuf** [comment]
Yea, I do actually have the same structure myself but I still encounter the error.

**Comment by nhs-webleym** [comment]
No. I had difficulties tho. Then I copied the structure and code from the commit and it worked fine.

**Comment by QKKQQK** [comment]
Check the #section head and make sure you typed the script src correctly, if there is a typo Handlebars will be undefined.
